lib/beaker/hypervisor/aws_sdk.rb

```diff
@@ -405,7 +400,12 @@ def launch_all_nodes
           instance = create_instance(host, ami_spec, nil)
           instances.push({:instance => instance, :host => host})
         end
+        # Add metadata tags to each instance
```

Comment:
@hunner from my testing, I'm inclined to think this will shave off ~30 seconds of time; is that sufficient for your needs? I recall in hipchat that we were looking at 20 minute long provision times, which still wouldn't be quite long enough before the reaper came and deleted the instance. 

A different solution might be to add the tags to the `config` object that is passed to `region.instances.create`  in the `create_instance` method. That would ensure that the tags are passed during instantiation. Have you tried to refactor the code into that method?